DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni

```diff
@@ -121,9 +121,9 @@
 #string STR_SLEEP_TYPE_S3     #language en-US "Suspend to RAM (S3)"
 #string STR_SLEEP_TYPE_S3_EXPERIMENTAL     #language en-US "Suspend to RAM (S3) - Experimental"
 
-#string STR_POWER_FAILURE_STATE_PROMPT  #language en-US "Power state after power failure"
-#string STR_POWER_FAILURE_STATE_HELP    #language en-US "Selects state the system gets into when power supply is restored after power failure.\n\n"
-                                                          "Note that the definition of power failure varies between platforms and it might take absence of power for up to 30 seconds for this option to apply."
+#string STR_POWER_FAILURE_STATE_PROMPT  #language en-US "Power state when connecting power supply after power failure"

```

Comment:
Do we really need this? The help explains what this options is about

Reply:
it does explain. But then again: https://github.com/Dasharo/dasharo-issues/issues/1021#issuecomment-2304440260

Reply:
```suggestion
#string STR_POWER_FAILURE_STATE_PROMPT  #language en-US "Power state after power/AC loss"
```

Does it sound more precise?

Reply:
it is shorter and that's good enough for me (seriously my version took 3 or 4 lines in setup menu, it was ugly)